common/src/lib.rs

```diff
@@ -146,6 +148,9 @@ pub enum IntermediateMessage {
         value: u128,
         reply: Option<H256>,
     },
+    WakeMessages {
```

Comment:
Is there a reason for this variant name to be a plural?

Reply:
No real reason. I thought about it like a command: "please wake messages addressed to this program". Anyway it't not hard to rename it. 'WakeMessage' or just 'Wake'?

Reply:
Whatever it is - as long as it's consistent, it's fine by me. Personally, I would give up the `...Message` ending part at all (going ahead with just `Init`, `Dispatch` and `Wake`). But it's up to you.